examples/integrations/oci/atp-reactive/pom.xml

```diff
@@ -60,8 +60,14 @@
             <artifactId>helidon-config-yaml</artifactId>
         </dependency>
         <dependency>
-            <groupId>io.helidon.integrations.oci</groupId>
-            <artifactId>helidon-integrations-oci-atp</artifactId>
+            <groupId>com.oracle.oci.sdk</groupId>
+            <artifactId>oci-java-sdk-database</artifactId>
+        </dependency>
+        <dependency>
+            <!-- required by SDK -->
```

Comment:
I'm not sure this is right.  It should be pulled in transitively.  My guess is this is something added to make `jakarta.*` imports work; they shouldn't be here either (see above).

Reply:
As you said in another comment, jakarta.inject still have classes defined with javax.inject package name. OCI SDK required this to be available at runtime else it fails with `Caused by: java.lang.ClassNotFoundException: javax.inject.Singleton`

Since jakarta.inject is version managed in helidon's dependency pom, using it here. None of the test class is importing/using any jakarta.* classes.